modules/imgproc/perf/perf_integral.cpp

```diff
@@ -5,9 +5,22 @@
 
 namespace opencv_test {
 
+enum PerfSqMatDepth{
+    SZ_32S32S = 0,
+    SZ_32S32F,
+    SZ_32S64F,
+    SZ_32F32F,
+    SZ_32F64F,
+    SZ_64F64F};
+
+CV_ENUM(IntegralOutputDepths, SZ_32S32S, SZ_32S32F, SZ_32S64F, SZ_32F32F, SZ_32F64F, SZ_64F64F);
```

Comment:
> SZ_

What is the meaning of this?
Depth != Size (size is usually associated with image size in pixels, not a "size" of type)

`DEPTH_32S_32S`?